challenges/3-repeating/repeating.js

```diff
@@ -3,7 +3,11 @@
     For example, `repeating("I'm the best", 3)` should console.log "I'm the best" 3 times.
     You will need to use a loop.
 */
-
-function repeating(message, times) {}
+function repeating(message, times) {
+    let i = 0;
+    for (i = 0; i < times; i++) {
```

Comment:
instead of this, do `for(let i = 0; i < times; i++)` this has the advantage of keeping that i variable's scope isolated to just the loop

Reply:
^ and make sure you edit all the loops for it

Reply:
I rewrote the function 
`function repeating(message, times) { 
      for (i = 0; i < times; i++) {
           console.log(message); 
      } 
} `
but it gave me an error `ReferenceError: i is not defined`. Would I need to declare `let i = 0` outside of the function? Or are you saying I don't need to declare i at all? If so, how would I get past this reference error? 
<img width="1440" alt="repeating-js" src="https://user-images.githubusercontent.com/62922013/79822648-46b6cd00-8357-11ea-83b1-e8d6d31476de.png">

Reply:
Ignore my comments!! I overlooked `let i = 0` in the for loop statement that you wrote out.